overlord/configstate/configcore/corecfg.go

```diff
@@ -57,6 +58,11 @@ func Run(tr Conf) error {
 		return err
 	}
 
+	// capture cloud information
+	if err := handleCloud(tr); err != nil {
```

Comment:
(nitpick) I wonder if a more specific name is more appropriate here given that this function only acts when seeding. Maybe something like `setCloudConfWhenSeeding` or similar?

Reply:
using setCloudInfoWhenSeeding